CHANGELOG.md

```diff
@@ -11,6 +11,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ### Changed
 
+ * [Core] Refactor CommandLine arguments ([2130](https://github.com/cucumber/cucumber-jvm/pull/2130) Quang Lê)
```

Comment:
The change log is read from the perspective of a user of the library. From that perspective you did not refactor any thing. Rather you added constants for use with the CLI.

Reply:
yeah I've just updated it for easier to understand 👍